drivers/util.py

```diff
@@ -130,9 +130,9 @@ def _getDateString():
     return "%s-%s-%s:%s:%s:%s" % \
           (t[0],t[1],t[2],t[3],t[4],t[5])
 
-def doexec(args, inputtext=None):
+def doexec(args, inputtext=None, new_env={}):
     """Execute a subprocess, then return its return code, stdout and stderr"""
-    proc = subprocess.Popen(args,stdin=subprocess.PIPE,stdout=subprocess.PIPE,stderr=subprocess.PIPE,close_fds=True)
+    proc = subprocess.Popen(args,stdin=subprocess.PIPE,stdout=subprocess.PIPE,stderr=subprocess.PIPE,close_fds=True, env=dict(os.environ.items() + new_env.items()))
```

Comment:
Just `env=dict(os.environ, new_env)` I think

Reply:
Fixed

Reply:
That did not work, unfortunately, I had to use .update